Deck of Cards:
 Write a program with exception handling that uses try, catch, and finally.
 Catch both a standard exception and a custom exception.

CCSF Exercise #22:
  Out of Range Exception and Throw Exception Tester .java files.
  Write a main method that will create a Scanner, and use the method getIntegers() to get a bunch of integers from the user. 
  Once the list is returned, just print out the list in the main method.
  Don't include any exception handling--just throw the exception in the main method.
  Now try running it and see what happens if you have an out of bounds value.
  Now, rewrite the main method using try-catch block to prompt the user to re-enter any number that is out of bounds.
